### ISSUE TITLE
SEBLOD and MultiSite + MySQL 8.0.x error in SQL syntax

### ISSUE THREAD

**Issue by DenysNosov** [Octopoos/SEBLOD#589]
Hi!

I'm enable MultiSite in SEBLOD and get MySQL error in all my site (administrator and front site):

```
1064 You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'groups, public_viewlevel, viewlevels, configuration, options FROM #__cck_core' at line 1
```

Clean install Joomla! 3.9.14 or 15
Clean install SEBLOD 3.18.0 without external SEBLOD plugins

**Comment by sebastienheraud** [comment]
Hi Denys,
wip: https://github.com/Octopoos/SEBLOD/commit/cf259e3035e16972a197ae705c369e9c7b807dd5
ref: https://www.seblod.com/community/forums/general-discussions/enabling-multi-site-corrupt-joomla-fresh-install#post64905
Saba

**Comment by sebastienheraud** [comment]
Hi,
This should be fixed with SEBLOD 3.19.
Feel free to re-open if there is any issue.
Saba
